HACKING.md

```diff
@@ -45,14 +45,14 @@ The following should work in a fresh Git clone:
       sudo dnf builddep --spec $TEMPFILE && \
       rm $TEMPFILE
 
-In addition, for testing, the following dependencies are required:
-
-    $ sudo dnf install curl expect \
-        libvirt libvirt-client libvirt-daemon libvirt-python \
-        python python-libguestfs python-lxml libguestfs-xfs \
-        python3 libvirt-python3 \
-        libguestfs-tools qemu qemu-kvm rpm-build rsync xz \
-        chromium-headless
+For running integration tests, the following dependencies are required:
+
+    $ sudo dnf install curl expect xz rpm-build chromium-headless \
+        libvirt-daemon-kvm libvirt-client python3-libvirt \
```

Comment:
Any reason for this trailing slash or just forgotten?

Reply:
Just forgotten, thanks for spotting! Fixed.